compiler/modules/front/const.bal

```diff
@@ -556,10 +583,14 @@ function expectsFloat(t:SemType? semType) returns boolean {
     }
 }
 
-function floatFromDecimalLiteral(string digits) returns float|error {
+function floatFromFpLiteral(string digits) returns float|error {
     return float:fromString(digits);
```

Comment:
Please verify that error case will never occur and then change the code as follows
```ballerina
function floatFromDecimalLiteral(string digits) returns float {
  float|error f = float:fromString(digits);
  if f is error {
      panic err:impossible();
  }
  return f;
}
```
And lets add a comment before method saying the prerequisites.

Reply:
Changed